### PR TITLE
Debian packaging improvements

### PR DESCRIPTION
**Pull request by pktiuk**
Everything is done according to Debian guidelines and  informations printed by `lintian`

lintian report for currently available packages:
```
E: xournalpp: debian-changelog-file-missing
E: xournalpp: extended-description-is-empty
E: xournalpp: missing-dependency-on-libc needed by usr/bin/xournal-thumbnailer and 1 others
E: xournalpp: no-copyright-file
E: xournalpp: unstripped-binary-or-object usr/bin/xournalpp

W: xournalpp: binary-without-manpage usr/bin/xournal-thumbnailer
W: xournalpp: binary-without-manpage usr/bin/xournalpp
W: xournalpp: script-not-executable usr/share/xournalpp/ui/icons/hicolor/update-icon-cache.sh
W: xournalpp: script-not-executable usr/share/xournalpp/ui/iconsDark/hicolor/update-icon-cache.sh
```
lintian report after these patches:
```
E: xournalpp: latest-changelog-entry-without-new-date
E: xournalpp: no-copyright-file
E: xournalpp: unstripped-binary-or-object usr/bin/xournalpp

W: xournalpp: binary-without-manpage usr/bin/xournalpp
W: xournalpp: binary-without-manpage usr/bin/xournalpp-thumbnailer
W: xournalpp: non-standard-dir-perm usr/ 0775 != 0755
W: xournalpp: non-standard-dir-perm usr/bin/ 0775 != 0755
W: xournalpp: non-standard-dir-perm usr/share/ 0775 != 0755
W: xournalpp: non-standard-dir-perm ... use --no-tag-display-limit to see all (or pipe to a file/program)
W: xournalpp: script-not-executable usr/share/xournalpp/ui/icons/hicolor/update-icon-cache.sh
W: xournalpp: script-not-executable usr/share/xournalpp/ui/iconsDark/hicolor/update-icon-cache.sh
```

### CODE REVIEW

**Comment by pktiuk** [comment]
I haven't added stripping binaries, because I am not sure yet if it would not mess with compiling with debugging flags.

Currently there is also error with changelog format (every single change from there was made in the same time) and it should be fixed later (I mean /debian/changelog should be rewritten).

File: CMakeLists.txt [hunk 1]
```diff
@@ -336,6 +336,35 @@ endif (CMAKE_DEBUG_INCLUDES_LDFLAGS)
 set (CPACK_OUTPUT_FILE_PREFIX packages)
 set (CPACK_PACKAGE_NAME "${PROJECT_PACKAGE}")
 set (CPACK_PACKAGE_DESCRIPTION_SUMMARY "Xournal++ - Open source hand note-taking program")
+set (CPACK_DEBIAN_PACKAGE_DESCRIPTION
```
Comment (LittleHuba) [inline CMakeLists.txt:339]: Is it possible to outsource this into a different file? I'm not quit happy to add this into the main CMakeLists.

Comment (pktiuk) [reply CMakeLists.txt:339]: I will try to do it

Comment (pktiuk) [reply CMakeLists.txt:339]: Where should be this description placed?

Comment (LittleHuba) [reply CMakeLists.txt:339]: Best would be to place this file somewhere with the docs.

Comment (pktiuk) [reply CMakeLists.txt]: Done
Now full package description is stored in `debian/package_description`

**Comment by Technius** [comment]
Looks good to me so far. Do you still want to update the changelog?

**Comment by pktiuk** [comment]
I could add this file do debian package, but I would leave adding this file up to you (or just pasting it here in comment), because I don't know history of this project.  
Useful links:
https://dep-team.pages.debian.net/deps/dep5/  
https://stackoverflow.com/questions/24788293/packaging-debian-files-debian-copyright-file-format

**Comment by Technius** [comment]
Alright, then I'll merge this now. Thanks for the PR!

**Pull request merged**